app/javascript/react/components/Users/UserPage.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from "react"
+
+const UserPage = (props) => {
+  const [user, setUser] = useState([])
+  const [profilePhoto, setProfilePhoto] = useState([])
```

Comment:
I know they are nested differently, but can we try to combine these into one piece of state?